manifests/install.pp

```diff
@@ -192,7 +192,7 @@
   # Render the Shibboleth configuration. These are run time and not used
   # during the build process. It should restart Jetty, though.
   ['ldap.properties', 'idp.properties', 'authn/general-authn.xml', 'logback.xml',
-    'c14n/subject-c14n.xml', 'saml-nameid.xml',
+    'c14n/subject-c14n.xml', 'saml-nameid.xml', 'access-control.xml',
```

Comment:
When you inherit then you get all the variables of the class you inherit, even in templates.
You may not want to rely on this, however and using inherit for anything but the params pattern and maybe roles is generally frowned upon.  (but I still see it in modules used in the way you are using it...)